openpype/hosts/houdini/api/lib.py

```diff
@@ -581,3 +582,74 @@ def splitext(name, allowed_multidot_extensions):
             return name[:-len(ext)], ext
 
     return os.path.splitext(name)
+
+
+def get_top_referenced_parm(parm):
+
+    processed = set()  # disallow infinite loop
+    while True:
+        if parm.path() in processed:
+            raise RuntimeError("Parameter references result in cycle.")
+
+        processed.add(parm.path())
+
+        ref = parm.getReferencedParm()
+        if ref.path() == parm.path():
+            # It returns itself when it doesn't reference
+            # another parameter
+            return ref
+        else:
+            parm = ref
+
+
+def evalParmNoFrame(node, parm, pad_character="#"):
+
+    parameter = node.parm(parm)
+    assert parameter, "Parameter does not exist: %s.%s" % (node, parm)
+
+    # If the parameter has a parameter reference, then get that
+    # parameter instead as otherwise `unexpandedString()` fails.
+    parameter = get_top_referenced_parm(parameter)
+
+    # Substitute out the frame numbering with padded characters
+    try:
+        raw = parameter.unexpandedString()
+    except hou.Error as exc:
+        print("Failed: %s" % parameter)
+        raise RuntimeError(exc)
+
+    def replace(match):
+        padding = 1
+        n = match.group(2)
+        if n and int(n):
+            padding = int(n)
+        return pad_character * padding
+
+    expression = re.sub(r"(\$F([0-9]*))", replace, raw)
+
+    with hou.ScriptEvalContext(parameter):
+        return hou.expandStringAtFrame(expression, 0)
+
+
+def get_color_management_preferences():
+    """Get default OCIO preferences"""
+    data = {
+        "config": hou.Color.ocio_configPath()
+
+    }
+
+    # Get default display and view from OCIO
+    display = hou.Color.ocio_defaultDisplay()
+    disp_regex = re.compile(r"^(?P<name>.+-)(?P<display>.+)$")
+    disp_match = disp_regex.match(display)
+
+    view = hou.Color.ocio_defaultView()
+    view_regex = re.compile(r"^(?P<name>.+- )(?P<view>.+)$")
+    view_match = view_regex.match(view)
+    data.update({
+        "display": disp_match.group("display"),
+        "view": view_match.group("view")
+
+    })
```

Comment:
Are we sure this is is correct?

I get this by default from Houdini (without custom OCIO set):
```python
>>> hou.Color.ocio_defaultDisplay()
'Display - sRGB'
>>> hou.Color.ocio_defaultView()
'Output - SDR Video - ACES 1.0'
```
These are the exact names from the `.ocio` config file and should thus NOT be stripped. Right?

Once I point it to e.g. aces1.0.3 config I get this:
```python
>>> hou.Color.ocio_defaultDisplay()
'ACES'
>>> hou.Color.ocio_defaultView()
'sRGB'
```

As such, I'm pretty sure this logic in this PR is wrong. Right @antirotor @moonyuet ?

Reply:
Seeing this reported on Discord [here](https://discord.com/channels/517362899170230292/611571347058982927/1123885523207979068) which might be related to what I mentioned above. @moonyuet @antirotor any chance any of you can confirm?

![image](https://github.com/ynput/OpenPype/assets/2439881/81e521b7-bd57-46d6-bc79-bdf118f36a5f)